### PR TITLE
aina story subplot

### PR DESCRIPTION
**Pull request by allister-translator**

### CODE REVIEW

File: Translated/Story/08-09 Finding Bruder/msq_011501.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+msq_011501_0010#0,"""We've got to find something that'll serve as the headstone first. When we get of this cave, let's check towards the forest."""
```
Comment (DabirSA) [inline Translated/Story/08-09 Finding Bruder/msq_011501.csv]: "When we get **out of** this cave."

And I don't like "check towards". "check around" or "head towards", maybe?

File: Translated/Story/08-09 Finding Bruder/msq_011503.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+msq_011503_0010#0,"""And of course, there's no shortage of enemies."""
+msq_011503_0020#0,"""I doubt they'll just let us look around in peace, so let's take down a few first."""
```
Comment (DabirSA) [inline Translated/Story/08-09 Finding Bruder/msq_011503.csv]: "let's take a few down" sounds more natural.

File: Translated/Story/08-09 Finding Bruder/msq_011512.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+msq_011512_0010#0,"""This area wasn't hit as hard as the rest..."""
+msq_011512_0020#0,"""Ah, that flower's so pretty... I wanna take some with me. Just a little bit."""
```
Comment (DabirSA) [inline Translated/Story/08-09 Finding Bruder/msq_011512.csv]: "Just a few"?

File: Translated/Story/08-09 Finding Bruder/ssc_011060.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+ssc_011060_0010#0,"""Emergency Alert: We have detected strong DOLLS energy readings in the Aerio Region. All ARKS Members, please prepare to sortie."""
+ssc_011060_0020#0,"""Patching through. Can you read me?"""
+ssc_011060_0030#0,"""As you just heard, we've detected a strong DOLLS energy signal. Their target appears to be the ARKS Mining Rigs."""
+ssc_011060_0040#0,"""As its name suggests, it is a type of heavy machinery that we use to excavate resources from underground."""
```
Comment (DabirSA) [inline Translated/Story/08-09 Finding Bruder/ssc_011060.csv]: You've switched from plural to singular here.

File: Translated/Story/08-09 Finding Bruder/ssc_011060.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+ssc_011060_0050#0,"""These excavated resources are necessary for City maintenance as well as the manufacturing of issued ARKS equipment."""
+ssc_011060_0055#0,"""Without these resources, we will not be able to continue fighting the DOLLS."""
+ssc_011060_0060#0,"""Radar equipment is being operated to anticipate DOLLS movement, but it's not uncommon for the radar itself to draw their attention in the same way."""
+ssc_011060_0070#0,"""The rigs are being escorted by resource convoy, but this is a large-scale offensive, so we anticipate a grueling battle."""
```
Comment (DabirSA) [inline Translated/Story/08-09 Finding Bruder/ssc_011060.csv]: Convoys? Or A convoy?

**Review by DabirSA — CHANGES_REQUESTED** [review]

File: Translated/Story/08-09 Finding Bruder/ssc_011060.csv (hunk 5 above)
Comment (allister-translator) [reply Translated/Story/08-09 Finding Bruder/ssc_011060.csv]: i have absolutely no idea so i fixed it with an english cheat code

File: Translated/Story/08-09 Finding Bruder/msq_011512.csv (hunk 3 above)
Comment (allister-translator) [reply Translated/Story/08-09 Finding Bruder/msq_011512.csv]: did the quest, fixed context

File: Translated/Story/08-09 Finding Bruder/msq_011503.csv (hunk 2 above)
Comment (allister-translator) [reply Translated/Story/08-09 Finding Bruder/msq_011503.csv]: did quest, fixed context

File: Translated/Story/08-09 Finding Bruder/msq_011501.csv (hunk 1 above)
Comment (allister-translator) [reply Translated/Story/08-09 Finding Bruder/msq_011501.csv]: we never even enter a cave during this quest so i honestly dont know where this file is even used. fixed anyways lol

**Review by DabirSA — APPROVED** [review]

**Pull request merged**